app/src/main/java/com/li/weatherapp/data/model/Components.kt

```diff
@@ -0,0 +1,29 @@
+package com.li.weatherapp.data.model
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+import org.json.JSONObject
+
+@Parcelize
+data class Components(
```

Comment:
Đặt tên cụ thể hơn là chung chung Components
```suggestion
data class AirComponents(
```